services/slack/intermediate_test.go

```diff
@@ -531,6 +531,26 @@ func TestIntermediateUserSanitise(t *testing.T) {
 		require.Equal(t, expectedEmail, user.Email)
 		require.Equal(t, -1, exitCode)
 	})
+
+	t.Run("Properties should respect the max length", func(t *testing.T) {
```

Comment:
We could use the `model.XMaxRunes` constants here as well, to ensure the test aligns with the values set in Mattermost in the future.